fx_crash_sig/cmd_get_crash_sig.py

```diff
@@ -2,39 +2,39 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from __future__ import print_function
-
 import argparse
 import sys
 
 import ujson as json
 
 from fx_crash_sig.crash_processor import CrashProcessor
 
+
 DESCRIPTION = """
 Takes raw crash trace and symbolicates it to return the crash signature
 """
 
 
 def cmdline():
     parser = argparse.ArgumentParser(description=DESCRIPTION)
-    parser.add_argument('-v', '--verbose', action='store_true')
-    parser.add_argument('-w', '--windows', action='store_true')
```

Comment:
Whether or not something is "Windows" or not only affects signature generation and it can be determined from the `normalized_os` field, so I took this argument out.